chromium/rules.js

```diff
@@ -250,20 +250,20 @@ RuleSets.prototype = {
 
     // Read user prefs
     if (rule_set.name in this.ruleActiveStates) {
-      rule_set.active = (this.ruleActiveStates[rule_set.name] == "true");
+      rule_set.active = this.ruleActiveStates[rule_set.name];
     }
 
     var rules = ruletag["rule"];
     for (let rule of rules) {
-      if (rule["from"] != null && rule["to"] != null) {
+      if (rule["from"] && rule["to"]) {
         rule_set.rules.push(new Rule(rule["from"], rule["to"]));
       }
     }
 
     var exclusions = ruletag["exclusion"];
-    if (exclusions != null) {
+    if (exclusions) {
       for (let exclusion of exclusions) {
-        if (exclusion != null) {
+        if (exclusion) {
```

Comment:
likewise, if `exclusion  = false`.
```javascript
exclusion != null // true
exclusion  // false
```

Reply:
Can `exclusion` be false?

Reply:
I don't know, I'm asking you. What can `exclusion` be? Is it possible for it to be falsey, but not null?

Reply:
I think it may be a string. In this case empty string is also falsey, but exclusions shouldn't be empty strings.